events/scheme_events/hostile_scheme_discovery_events.txt

```diff
@@ -547,16 +559,20 @@ hostile_scheme_discovery.3003 = {
 
 	option = {
 		name = hostile_scheme_discovery.3003.a
+		
+		# Warcraft
+		hostile_scheme_opinion_effect = {
+			VICTIM = scope:target
+			ATTACKER = scope:owner
+			SCHEME = scope:scheme.scheme_type
```

Comment:
`scheme_type` can't be accessed like this.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/118822769-7cffac80-b8c9-11eb-8db8-b09ce7dbab5c.png)

</details>

If you have an idea how to fix it, tell it to me before implementing it.